### PR TITLE
Update docs and miscellaneous

### PR DESCRIPTION
**Pull request by JaydenLiang**

### CODE REVIEW

**Review by jamie-pate — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,11 +1,41 @@
-# Autoscale AWS Library
+# FortiGate Autoscale on AWS
 
-This is an intial readme template.
+The implementation for FortiGate Autoscale using [Fortinet Autoscale](https://github.com/fortinet/autoscale-core) on Amazon AWS platfrom.
```
Comment (JaydenLiang) [inline README.md:3]:
```suggestion
The implementation for FortiGate Autoscale using Fortinet [Autoscale Core](https://github.com/fortinet/autoscale-core) on Amazon AWS platfrom.
```
@klc178 please review this change.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,41 @@
...
+  * or, manually generate the deployment package:
+    1. Chechkout the **main** branch of the project.
+    2. Run `npm run build-artifacts` at the project root directory.
+    3. Deployment packages **aws-cloudformation.zip** will be available in the **dist/artifacts** directory.
+
+## Deployment guides
+
+Deployment guides are available from the Fortinet Document Library:
+
+  + [<NAME_PLACE_HOLDER>](<LINK_PLACE_HOLDER>)
```
Comment (JaydenLiang) [inline README.md:27]: @klc178 please update it here with the doc reference to the official documentation.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,41 @@
...
 
 # Support
 Fortinet-provided scripts in this and other GitHub projects do not fall under the regular Fortinet technical support scope and are not supported by FortiCare Support Services.
-For direct issues, please refer to the [Issues](https://github.com/fortinet/autoscale-aws/issues) tab of this GitHub project.
+For direct issues, please refer to the [Issues](https://github.com/fortinet/fortigate-autoscale/issues) tab of this GitHub project.
```
Comment (JaydenLiang) [inline README.md:37]:
```suggestion
For direct issues, please refer to the [Issues](https://github.com/fortinet/fortigate-autoscale-aws/issues) tab of this GitHub project.
```
@klc178 please confirm.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,41 @@
...
 For other questions related to this project, contact [github@fortinet.com](mailto:github@fortinet.com).
 
 ## License
-[License](https://github.com/fortinet/autoscale-aws/blob/master/LICENSE) © Fortinet Technologies. All rights reserved.
+[License](https://github.com/fortinet/fortigate-autoscale/blob/master/LICENSE) © Fortinet Technologies. All rights reserved.
```
Comment (JaydenLiang) [inline README.md:41]:
```suggestion
[License](https://github.com/fortinet/fortigate-autoscale-aws/blob/master/LICENSE) © Fortinet Technologies. All rights reserved.
```
@klc178 please confirm.

File: README.md (hunk 1 above)
Comment (klc178) [reply README.md:3]:
```suggestion
An implementation for FortiGate Autoscale for the AWS SDK platform API with a Dynamo DB storage backend. This implementation requires Fortinet [Autoscale Core](https://github.com/fortinet/autoscale-core).
```

File: README.md [hunk 5]
```diff
@@ -1,11 +1,41 @@
-# Autoscale AWS Library
+# FortiGate Autoscale on AWS
 
-This is an intial readme template.
+The implementation for FortiGate Autoscale using [Fortinet Autoscale](https://github.com/fortinet/autoscale-core) on Amazon AWS platfrom.
+
+This project provides a multi-group Auto Scaling functionality for Fortinet FortiGate EC2 instances to form an HA cluster with failover protection.
```
Comment (klc178) [inline README.md:5]:
```suggestion
This project provides multi-group Auto Scaling functionality for Fortinet FortiGate EC2 instances to form an HA cluster with failover protection.
```

File: README.md [hunk 6]
```diff
@@ -1,11 +1,41 @@
...
+
+This project has the following features:
+
+1. Multi-group Hybrid Licensing models:
```
Comment (klc178) [inline README.md:9]:
```suggestion
1. Multi-group Licensing models:
```

File: README.md [hunk 7]
```diff
@@ -1,11 +1,41 @@
...
+   1. **Hybrid**: 1 fix-sized Auto Scaling group of 2 (and more) BYOL FortiGate Instances, and 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate Instances.
```
Comment (klc178) [inline README.md:10]:
```suggestion
   1. **BYOL-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) Bring Your Own Licence (BYOL) FortiGate instances.
```

File: README.md [hunk 8]
```diff
@@ -1,11 +1,41 @@
...
+   2. **BYOL-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) BYOL FortiGate Instances.
```
Comment (klc178) [inline README.md:11]:
```suggestion
   3. **PAYG-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate instances.
```

File: README.md [hunk 9]
```diff
@@ -1,11 +1,41 @@
...
+   3. **PAYG-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate Instances.
```
Comment (klc178) [inline README.md:12]:
```suggestion
   1. **Hybrid**: 1 fix-sized Auto Scaling group of 2 (and more) BYOL FortiGate instances, and 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate instances.
```

File: README.md [hunk 10]
```diff
@@ -1,11 +1,41 @@
...
+2. AWS Transit Gateway Integration.
+
+## Deployment packages
+
```
Comment (klc178) [inline README.md:16]:
```suggestion
To generate local deployment packages:
```

File: README.md [hunk 11]
```diff
@@ -1,11 +1,41 @@
...
+  * Download the **aws-cloudformation.zip** for the latest production version from [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases).
```
Comment (klc178) [inline README.md:17]:
```suggestion
  *  From the [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases), download the **aws-cloudformation.zip** for the latest production version.
```

File: README.md [hunk 12]
```diff
@@ -1,11 +1,41 @@
...
+  * or, manually generate the deployment package:
```
Comment (klc178) [inline README.md:18]:
```suggestion
  * Manually generate the deployment package:
```

File: README.md [hunk 13]
```diff
@@ -1,11 +1,41 @@
...
+    1. Chechkout the **main** branch of the project.
```
Comment (klc178) [inline README.md:19]:
```suggestion
    1. Check out the **main** branch of the project.
```

File: README.md [hunk 14]
```diff
@@ -1,11 +1,41 @@
...
+    2. Run `npm run build-artifacts` at the project root directory.
+    3. Deployment packages **aws-cloudformation.zip** will be available in the **dist/artifacts** directory.
```
Comment (klc178) [inline README.md:21]: ``suggestion
    3. The deployment package **aws-cloudformation.zip** will be available in the **dist/artifacts** directory.
```

File: README.md (hunk 2 above)
Comment (klc178) [reply README.md:27]:
```suggestion
  + [ FortiGate / FortiOS 6.2 Deploying auto scaling on AWS](https://docs.fortinet.com/vm/aws/fortigate/6.2/aws-cookbook/6.2.0/397979/deploying-auto-scaling-on-aws)
    + [ FortiGate / FortiOS 6.4 Deploying auto scaling on AWS](https://docs.fortinet.com/vm/aws/fortigate/6.4/aws-cookbook/6.4.0/397979/deploying-auto-scaling-on-aws)
```

Comment (klc178) [reply README.md:27]: are we supporting 6.4? It isn't in the drop down.

File: README.md [hunk 15]
```diff
@@ -1,11 +1,41 @@
...
+
+## Deployment guides
+
+Deployment guides are available from the Fortinet Document Library:
+
+  + [<NAME_PLACE_HOLDER>](<LINK_PLACE_HOLDER>)
+
+## Launch a demo
+
+| Non-TGW (new VPC) | Non-TGW (existing VPC) | TGW Integration (new VPC) |
```
Comment (klc178) [inline README.md:31]:
```suggestion
| New VPC, no-TGW | Eexisting VPC, no-TGW | TGW Integration (new VPC) |
```

File: README.md (hunk 3 above)
Comment (klc178) [reply README.md:37]: confirmed for:
For direct issues, please refer to the [Issues](https://github.com/fortinet/fortigate-autoscale-aws/issues) tab of this GitHub project.

File: README.md (hunk 4 above)
Comment (klc178) [reply README.md:41]: Confirmed:
[License](https://github.com/fortinet/fortigate-autoscale-aws/blob/master/LICENSE) © Fortinet Technologies. All rights reserved.

Comment (klc178) [reply README.md:41]: Please update the date in the license file to 2020.

**Review by klc178 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (JaydenLiang) [reply README.md:27]: > are we supporting 6.4? It isn't in the drop down.

Not in this version but in the plan to support it soon.

File: README.md (hunk 6 above)
Comment (JaydenLiang) [reply README.md:9]: as discussed, we will keep the Hybrid so we do no take this suggestion.

File: README.md (hunk 14 above)
Comment (JaydenLiang) [reply README.md:21]: accepted down here.

File: README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (klc178) [reply README.md:16]:
```suggestion
To obtain the deployment package, use one of the following:
```

File: README.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (klc178) [reply README.md:17]: I approve.
  * Download the package **aws-cloudformation.zip** directly from the [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases).

File: README.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (klc178) [reply README.md:18]: Confirmed:
  * Manually generate the deployment package in your local workspace:

File: README.md (hunk 13 above)
Comment (klc178) [reply README.md:19]: approved.
    1. From the [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases), download the source code (.zip or .tar.gz) for the latest version.

File: README.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (klc178) [reply README.md:21]: looks great! (steps 2 -5)

File: README.md [hunk 16]
```diff
@@ -1,11 +1,41 @@
...
+
+  * Download the **aws-cloudformation.zip** for the latest production version from [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases).
+  * or, manually generate the deployment package:
+    1. Chechkout the **main** branch of the project.
+    2. Run `npm run build-artifacts` at the project root directory.
+    3. Deployment packages **aws-cloudformation.zip** will be available in the **dist/artifacts** directory.
+
+## Deployment guides
+
+Deployment guides are available from the Fortinet Document Library:
```
Comment (klc178) [inline README.md:25]:
```suggestion
A deployment guide is available from the Fortinet Document Library:
```

File: README.md [hunk 17]
```diff
@@ -1,11 +1,41 @@
...
+
+## Deployment packages
+
+  * Download the **aws-cloudformation.zip** for the latest production version from [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases).
+  * or, manually generate the deployment package:
+    1. Chechkout the **main** branch of the project.
+    2. Run `npm run build-artifacts` at the project root directory.
+    3. Deployment packages **aws-cloudformation.zip** will be available in the **dist/artifacts** directory.
+
+## Deployment guides
```
Comment (klc178) [inline README.md:23]:
```suggestion
## Deployment guide
```

File: README.md [hunk 18]
```diff
@@ -1,11 +1,41 @@
...
+
+Deployment guides are available from the Fortinet Document Library:
+
+  + [ FortiGate / FortiOS 6.2 Deploying auto scaling on AWS](https://docs.fortinet.com/vm/aws/fortigate/6.2/aws-cookbook/6.2.0/397979/deploying-auto-scaling-on-aws)
+
+## Launch a demo
+
+| New VPC, no-TGW | Eexisting VPC, no-TGW | TGW Integration (new VPC) |
```
Comment (klc178) [inline README.md:31]:
```suggestion
| New VPC, no-TGW | Existing VPC, no-TGW | TGW Integration (new VPC) |
```

**Review by klc178 — CHANGES_REQUESTED** [review]
Overall, looks good. I found a typo and changes "guides" to "guide".

**Review by klc178 — APPROVED** [review]

**Pull request merged**